docs/src/wallet-guide/solflare.md

```diff
@@ -48,21 +48,24 @@ Keystore file, you will be taken to the Access page directly.
 You will be prompted to enter the password and upload your Keystore file,
 then you will be taken to the wallet interface main page.
 
-### Using a Ledger Nano S hardware wallet
+### Using a Ledger Nano hardware wallet
+
+*NOTE: SolFlare and Solana supports both Ledger Nano S and Nano X.
+There is a known issue in which a Nano X sometimes cannot connect using the
+Windows operating system. The Ledger team is working to resolve this.*
```

Comment:
I think this issue is going to affect any browser-based wallet using WebUSB

Reply:
Should the language be changed?  This is the only instance in the ecosystem that has this issue right now, and is unlikely to change for the time being, no?

Reply:
That's correct now.  Adding support should be fairly trivial though.  Sollet could probably add it fairly easily, the only barrier being their unfortunate choice of a non-BIP44, BIP32 derivation root.

I'm thinking it might make sense to put this in the main article and link out in this one

Reply:
what "main article"?

Reply:
Sorry, main _Ledger_ article.